<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixAndFluxClass</class>
 <widget class="QMainWindow" name="FixAndFluxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FixAndFlux</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionConnect_to_database"/>
    <addaction name="actionDisconnect_from_database"/>
    <addaction name="actionAdd_a_console"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_to_database"/>
   <addaction name="actionDisconnect_from_database"/>
   <addaction name="actionAdd_a_console"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect_to_database">
   <property name="icon">
    <iconset resource="FixAndFlux.qrc">
     <normaloff>:/icons/database.png</normaloff>:/icons/database.png</iconset>
   </property>
   <property name="text">
    <string>Connect to database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDisconnect_from_database">
   <property name="icon">
    <iconset resource="FixAndFlux.qrc">
     <normaloff>:/icons/disconnected.png</normaloff>:/icons/disconnected.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect from database</string>
   </property>
  </action>
  <action name="actionAdd_a_console">
   <property name="icon">
    <iconset resource="FixAndFlux.qrc">
     <normaloff>:/icons/add_console.png</normaloff>:/icons/add_console.png</iconset>
   </property>
   <property name="text">
    <string>Add a console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FixAndFlux.qrc"/>
 </resources>
 <connections/>
</ui>
